docrawl/docrawl_client.py

```diff
@@ -251,16 +265,16 @@ def get_current_url(self, filename, timeout=20):
 
         self._execute_function('get_current_url', inp, timeout)
 
-    def close_browser(self, timeout=10):
+    def close_browser(self, timeout=70):
```

Comment:
why timeout 70?

Reply:
Only because of my debugging, my WebDriver` needs 60 seconds to close a browser. Does not happen on Joaquin's mac.